tests/fixtures/parsers/typescript-parsers/new-parens.js

```diff
@@ -0,0 +1,262 @@
+"use strict";
+
+// new Storage<RootState>('state');
+
+exports.parse = () => ({
+    "type": "Program",
```

Comment:
Some of the files in `tests/fixtures` intentionally contain linting errors (in order to verify that ESLint works correctly on particular files), so I don't think unignoring the fixtures entirely would work. It might be better to only ignore certain fixtures, or to move code that is intended to be executed (rather than statically analyzed) out of the `tests/fixtures` directory. (That doesn't need to happen in this PR, though.)